verification/verify_confirm.py

```
@@ -1,69 +1,202 @@
+
+    with sync_playwright() as playwright:
+        browser = playwright.chromium.launch(headless=True)
+        try:
+            for scenario_name, scenario in scenarios:
+                print(f"[e2e] START: {scenario_name}")
+                context = browser.new_context()
+                try:
+                    page = context.new_page()
+                    scenario(page)
+                    print(f"[e2e] PASS: {scenario_name}")
+                except Exception as error:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)